src/HttpClient/HttpClientSmsUser.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\HttpClient;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Contracts\HttpClient\Exception\ClientExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\RedirectionExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\ServerExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class HttpClientSmsUser
+{
+    private HttpClientInterface $client;
+
+    public function __construct(HttpClientInterface $smsClient)
+    {
+        $this->client = $smsClient;
+    }
+
+    /**
+     * @throws TransportExceptionInterface
+     * @throws ServerExceptionInterface
+     * @throws RedirectionExceptionInterface
+     * @throws ClientExceptionInterface
+     */
+    public function sendSms(string $phone, string $message): void
+    {
+        $postData = [
+            'receiver' => $phone,
+            'body' => $message
+        ];
+        $postJson = json_encode($postData);
+
+        $this->client->request(
+            Request::METHOD_POST,
+            '',
```

Comment:
why empty string? It should be /api/messages or /messages

Reply:
Ok. Changed value in env.local